projects/models.py

```diff
@@ -160,7 +160,9 @@ def create_project(self, name, owner, description, status="active", project_temp
         key = self.generate_passkey()
         letters = string.ascii_lowercase
         secret = self.generate_passkey(40)
-        slug = slugify(name)
+        slug = slugify(name[:47])
+        if len(slug) < 3:
+            slug = "".join(random.choice(letters) for i in range(3))
```

Comment:
So if they name their project ML or something short, the slug will be "tgz-eog". Seems a bit arbitrary, but i guess it works. Maybe we chould just have random slugs?